architecture/azure/data-stores.md

```diff
@@ -0,0 +1,142 @@
+
+## Azure SQL Database
+
+[Azure SQL Database](https://azure.microsoft.com/en-us/products/azure-sql/database/) is a fully managed database service based on the [Microsoft SQL Server database engine](https://learn.microsoft.com/en-us/sql/sql-server/?view=sql-server-ver16). 
+
+:heavy_plus_sign: Pros:
+
+- Fully managed
+- Rich built-in scaling options
+- Automatic backups and point-in-time restores
+- Active Geo-replication and zone-redundancy
+- Different pricing tiers for more cost control
+
+:heavy_minus_sign: Cons:
+
+- Migrating existing SQL server instances might be challenging
+
+Azure SQL Database is supported via the [NServiceBus SQL persistence package](/persistence/sql/).
+
+[**Try the NServiceBus SQL persistence sample →**](/samples/sql-persistence/simple/)
+
+## Azure SQL Managed Instance
+
+[Azure SQL Managed Instance](https://azure.microsoft.com/en-us/products/azure-sql/managed-instance) is a fully managed database service that provides a near-complete compatibility with the [Microsoft SQL Server database engine](https://learn.microsoft.com/en-us/sql/sql-server/?view=sql-server-ver16). It offers a managed solution for migrating and modernizing existing SQL Server applications with minimal code changes.
+
+:heavy_plus_sign: Pros:
+
+- High degress of compatibility with SQL Server
+- Supports cross-database queries
+- Predictable performance for large databases due to isolation
+- More database configuration options compared to Azure SQL Database
+
+:heavy_minus_sign: Cons:
+
+- Higher costs compared to manually managed database servers or Azure SQL Database
+
+Azure SQL Managed Instances is supported via the [NServiceBus SQL persistence package](/persistence/sql/).
+
+[**Try the NServiceBus SQL persistence sample →**](/samples/sql-persistence/simple/)
+
+## Other data store options
+
+Azure provides further relational database services compatible with NServiceBus, e.g. [Azure Database for MySQL](https://azure.microsoft.com/en-us/products/mysql) and [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/products/postgresql).
+
+While the presented storage options provide the best experience and integration with Azure, other storage options are available and may be valid options depending on the specific requirements. [Explore all supported data stores](/persistence/#supported-persisters).
+
+## Choosing a data store
+
+The following factors should be considered when choosing a data store:
+
+### Existing infrastructure
+
+Using existing infrastructure and know-how can significantly speed up the development process and lower the risks associated with using unfamiliar data store technologies. Unless the existing infrastructure and technologies do not meet the architectural requirements, they are a good default choice.
+
+### Compatibility
+
+Data stores that are compatible with existing protocols and languages. For example, a data store may understand PostgreSQL, SQL Server, and MongoDB dialects.
```

Comment:
I'm not really sure what point I, as a reader, am supposed to take from this.

Reply:
@DavidBoike maybe something like this:

```suggestion

When migrating or modernizing existing applications, data stores that are compatible with currently used protocols and languages simplify the switch to cloud-based services significantly. For example, a data store may understand PostgreSQL, SQL Server, and MongoDB dialects which allow applications using any of these dialects to connect with minimal changes required.
```